languages.json

```diff
@@ -1432,6 +1432,14 @@
       "quotes": [["\\\"", "\\\""]],
       "extensions": ["sol"]
     },
+
+    "Spade": {
+      "line_comment": ["//"],
+      "multi_line_comments": [["/*", "*/"]],
+      "nested": true,
+      "important_syntax": ["///"],
```

Comment:
Why is this placed in the important syntax key?

Reply:
Since the syntax is very similar to Rust and it has that there. Both languages use `///` as doc comments, but maybe I misunderstood what that does